test/e2e/wmco_test.go

```diff
@@ -33,12 +36,15 @@ func TestWMCO(t *testing.T) {
 	gc.numberOfNodes = int32(numberOfNodes)
 	gc.skipNodeDeletion = skipNodeDeletion
 	gc.sshKeyPair = sshKeyPair
+	require.NotEmpty(t, privateKeyPath, "private-key-path is not set")
+	gc.privateKeyPath = privateKeyPath
 	// When the OPENSHIFT_CI env var is set to true, the test is running within CI
 	if inCI := os.Getenv("OPENSHIFT_CI"); inCI == "true" {
 		// In the CI container the WMCO binary will be found here
 		wmcoPath = "/usr/local/bin/windows-machine-config-operator"
 	}
 
+	t.Run("operator deployed", testOperatorDeployed)
```

Comment:
What is the reason for adding this test? Ideally this should be a separate commit or PR.

Reply:
The reasons is to make sure the operator can deploy without the secret already created, and then use a secret created after the operator is running

Reply:
Please add that as a comment as it is not obvious.

Reply:
`"operator deployed without private key secret"`